### ISSUE TITLE
[BUG] - Help of "cardano-cli query protocol-parameters" command shows "--epoch-slots" twice

### ISSUE THREAD

**Issue by vs4vijay** [input-output-hk/cardano-node#2639]
**Internal/External**
External

**Area**
Other

**Summary**
Help of "cardano-cli query protocol-parameters" command shows "--epoch-slots" twice.

**Steps to reproduce**
docker run -e --entrypoint cardano-cli -it inputoutput/cardano-node:latest query protocol-parameters

**Expected behavior**
"--epoch-slots" option should be listed only once.

**System info (please complete the following information):**
Docker Image:  inputoutput/cardano-node:1.26.2
cardano-cli 1.26.2 - linux-x86_64 - ghc-8.10
git rev 3531289c9f79eab7ac5d3272ce6e6821504fec4c

**Screenshots and attachments**
![image](https://user-images.githubusercontent.com/216637/115603587-fb365680-a2fd-11eb-8f0a-301b95309cec.png)

**Additional context**

**Comment by rdlrt** [comment]
Duplicate of #1451

**Comment by newhoggy** [comment]
The offending part of the code are here:

* https://github.com/input-output-hk/cardano-node/blob/master/cardano-submit-api/src/Cardano/TxSubmit/CLI/Parsers.hs#L94
* https://github.com/input-output-hk/cardano-node/blob/master/cardano-submit-api/src/Cardano/TxSubmit/CLI/Parsers.hs#L100

We use `optparse-applicative` to parse command line arguments and it does not appear to be any means to de-duplicate help.

I'm afraid this means we will have to mark this and won't fix until this changes.
